rosbag2_compression/src/rosbag2_compression/zstd_compressor.cpp

```diff
@@ -30,10 +30,17 @@ namespace
 //   - Increase the time taken to compress
 //   - Decrease the size of the compressed data
 // Setting to zero uses Zstd's default value of 3.
-constexpr const int DEFAULT_ZSTD_COMPRESSION_LEVEL = 1;
+constexpr const int kDefaultZstdCompressionLevel = 1;
```

Comment:
not that I really matters, but given that the style guide recommends `g_` followed by underscore, I think we could do the same for `k_` for constants.

Reply:
I interpreted the variables as being constants and not globals since its scope is restricted to the file due to being in an anonymous namespace.

Reply:
My understanding also is that global constants are `kCamelCase` and global variables are `g_snake_case`.